### PR TITLE
some updates

### PR DESCRIPTION
**Pull request by gearama**
added a couple of links to readme, and fixed some typos 
# Pull Request Checklist

Please leverage this checklist as a reminder to address commonly occurring feedback when submitting a pull request to make sure your PR can be reviewed quickly:

See the detailed list in the [contributing guide](https://github.com/Azure/azure-sdk-for-cpp/blob/main/CONTRIBUTING.md#pull-requests).

- [x] [C++ Guidelines](https://azure.github.io/azure-sdk/cpp_introduction.html)
- [x] Doxygen docs
- [x] Unit tests
- [x] No unwanted commits/changes
- [x] Descriptive title/description
  - [x] PR is single purpose
  - [x] Related issue listed
- [x] Comments in source
- [x] No typos
- [x] Update changelog
- [x] Not work-in-progress
- [x] External references or docs updated
- [x] Self review of PR done
- [x] Any breaking changes?

### CODE REVIEW

File: sdk/keyvault/azure-security-keyvault-secrets/test/samples/sample1-basic-operations/sample1-basic-operations.cpp [hunk 1]
```diff
@@ -2,11 +2,11 @@
 // SPDX-License-Identifier: MIT
 
 /**
- * @brief This sample provides the code implementation to use the Key Vault Secrets SDK client for
+ * @brief This sample provides the code implementation to use the KeyVault Secrets SDK client for
```
Comment (vhvb1989) [inline sdk/keyvault/azure-security-keyvault-secrets/test/samples/sample1-basic-operations/sample1-basic-operations.cpp:5]: We should follow `Key Vault` for any documentation as that's how the official documentation is:
https://docs.microsoft.com/en-us/azure/key-vault/

We can join it on the source, but comments/readme/docs should be `Key Vault`

Comment (gearama) [reply sdk/keyvault/azure-security-keyvault-secrets/test/samples/sample1-basic-operations/sample1-basic-operations.cpp]: done

**Review by vhvb1989 — APPROVED** [review]

File: sdk/keyvault/azure-security-keyvault-secrets/test/samples/sample1-basic-operations/sample1-basic-operations.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ secretClient.SetSecret(secretName, secretValue);
 
 ## Getting a Secret
 
-To get a secret from the keyvault  you will need to call GetSecret.
+To get a secret from the key vault  you will need to call GetSecret.
```
Comment (RickWinter) [inline sdk/keyvault/azure-security-keyvault-secrets/test/samples/sample1-basic-operations/sample1-basic-operations.md:40]:
```suggestion
Call GetSecret to retrieve a secret from Key Vault.
```

File: sdk/keyvault/azure-security-keyvault-secrets/test/samples/sample2-backup-restore/sample2-backup-restore.md [hunk 3]
```diff
@@ -37,7 +37,7 @@ secretClient.SetSecret(secretName, secretValue);
 
 ## Getting a Secret
 
-To get a secret from the keyvault  you will need to call GetSecret.
+To get a secret from the key vault  you will need to call GetSecret.
```
Comment (RickWinter) [inline sdk/keyvault/azure-security-keyvault-secrets/test/samples/sample2-backup-restore/sample2-backup-restore.md:40]:
```suggestion
Call GetSecret to retrieve a secret from Key Vault.
```

File: sdk/keyvault/azure-security-keyvault-secrets/test/samples/sample2-backup-restore/sample2-backup-restore.md [hunk 4]
```diff
@@ -48,13 +48,13 @@ std::cout << "Secret is returned with name " << secret.Name << " and value " <<
 
 ## Creating a Backup for the secret properties
 
-In order to get the backup of the secret we need to call BackupSecret, which will return a vector of bytes representing the backed up content. 
+In order to get the backup of the secret we need to call BackupSecret, which will return a vector of bytes representing the backed up content.
```
Comment (RickWinter) [inline sdk/keyvault/azure-security-keyvault-secrets/test/samples/sample2-backup-restore/sample2-backup-restore.md:51]:
```suggestion
Call BackupSecret to retrieve the secret backup.  BackupSecret will will return a vector of bytes representing the backed up content.
```

**Review by RickWinter — COMMENTED** [review]
Review all entries to standardize the descriptive statements.  Casing needs attention in some spots

**Pull request merged**